src/swaps/liquality/LiqualitySwapProvider.js

```diff
@@ -272,7 +273,11 @@ class LiqualitySwapProvider extends SwapProvider {
 
     try {
       const tx = await fromClient.chain.getTransactionByHash(swap.fromFundHash)
-
+      if (!tx && hasTimedOut(swap)) {
+        return {
+          status: 'REFUNDED'
```

Comment:
I guess that you should have 

endTime property here

similar to
          endTime: Date.now(),
          status: 'FAILED'
     
Might be important for the timeline diagram